### PR TITLE
Prevent the user to disable or close ssh access when only SSH key based authentication is used.

### PR DESCRIPTION
**Pull request by teclator**
## Problem

When the user selects to authenticate using only SSH key based authentication we propose to enable and open the SSH ports (since https://github.com/yast/yast-firewall/pull/103)

If the user change the SSH proposal, then the user might not be allowed to login.

- https://trello.com/c/Mr566lSn/2059-3-shouldhave-add-ssh-enable-disable-to-root-password-dialog

## Solution

Warn the user about the possible access problem

SSH_warning

### CODE REVIEW

File: package/yast2-firewall.changes [hunk 1]
```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Fri Oct  9 10:42:32 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Warn the user when SSH open port is closed or the the service is
+  disable and the configured authentication is only based on SSH
```
Comment (imobachgs) [inline package/yast2-firewall.changes:5]: disabled

**Review by imobachgs — APPROVED** [review]
Just a typo. Otherwise, LGTM.

File: package/yast2-firewall.changes [hunk 2]
```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Fri Oct  9 10:42:32 UTC 2020 - Knut Anderssen <kanderssen@suse.com>
+
+- Warn the user when SSH open port is closed or the the service is
```
Comment (lslezak) [inline package/yast2-firewall.changes:4]:
```suggestion
- Warn the user when the SSH port is closed or the service is
```

File: package/yast2-firewall.changes (hunk 1 above)
Comment (teclator) [inline package/yast2-firewall.changes:5]:
```suggestion
  disabled and the configured authentication is only based on SSH
```

**Pull request merged**